docs/deploying-airbyte/local-deployment.md

```diff
@@ -1,65 +1,39 @@
 # Local Deployment
+This page guides you through setting up your Airbyte on your local machine. 
 
-:::info
-These instructions have been tested on MacOS, Windows 10 and Ubuntu 20.04.
+## Prerequisites
+* [Install Docker](https://docs.docker.com/get-docker/) and make sure you have the latest version of `docker-compose` installed.
+* Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/)  and enable Hyper-V Windows Features".
 
-:::
+> **_Note:_** These instructions have been tested on MacOS, Windows 10, and Ubuntu 20.04.
 
-## Setup & launch Airbyte
+## Deploy on Mac and Linux
+To set up Airbyte on your local machine:
 
-- Install Docker on your workstation \(see [instructions](https://www.docker.com/products/docker-desktop)\). Make sure you're on the latest version of `docker-compose`.
-- After Docker is installed, you can immediately get started locally by running:
+1. Clone the Airbyte repository.
+2. Move into the cloned folder (`airbyte`) and start the server with `docker compose`;
 
 ```bash
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
-docker-compose up
+sudo docker compose up
```

Comment:
why the `sudo`? This is not necessary if you have docker added to your permission group.

Reply:
@Bam92 Please fix this